catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transformer/TransformerManager.java

```diff
@@ -53,6 +55,25 @@ public TransformerManager(List<ServiceReference> serviceReferences) {
         return getAvailableProperty(ID);
     }
 
+    public String getTransformerIdForSchema(String schema) {
+
+        List<Map<String, Object>> properties = getRelatedTransformerProperties(SCHEMA, schema);
+        if (properties.size() > 0) {
+            return (String) properties.get(0)
+                    .get(ID);
+        }
+        return null;
```

Comment:
Instead of returning null consider returning an empty string.  That should still satisfy comparisons and prevent NPEs.